spark-operator.yaml

```diff
@@ -26,15 +26,17 @@ pipeline:
       tag: spark-operator-chart-${{package.version}}
       expected-commit: bd2eda216b79733c992861f257b28dad36bb4c62
 
+  - runs: |
+      # Remediate GHSA-8mjg-8c8g-6h85 drop v1.19.5 to upgrade kubernetes version accordingly.
+      go mod edit -dropreplace k8s.io/kubernetes
```

Comment:
Doesn't `gobump` do that part automatically since `k8s.io/kubernetes@v1.25.16` is listed below?

Reply:
I tried with gobump but it does a smarter movement here:  It internally does `go mod edit -replace=k8s.io/kubernetes=k8s.io/kubernetes@v1.25.16`  and that results in an error:

```
go: k8s.io/kubernetes@v1.25.3 requires
	k8s.io/component-helpers@v0.0.0: reading k8s.io/component-helpers/go.mod at revision v0.0.0: unknown revision v0.0.0
```

When it should run `go get kubernetes@v1.25.16` and then `replace kubernetes=kubernetes@v1.25.16`. Until we change this behavior I need to drop the replace statement here.